src/docs/user/hcatalog.txt

```diff
@@ -61,6 +61,10 @@ The presence of the +--hcatalog-table+ option signifies that the import
 or export job is done using HCatalog tables, and it is a required option for
 HCatalog jobs.
 
++--hcatalog-external-table+::
```

Comment:
Using this option makes sense only together with --create-hcatalog-table thus could you please add fail fast scenarios to prevent separate usage? Also could you please update the documentation accordingly?
For reference please see SQOOP-816 and SQOOP-3324 where external Hive table option has been introduced and documented.

Reply:
Thanks for extending the documentation!